src/workerd/server/tests/container-client/test.js

```diff
@@ -46,15 +46,16 @@ export class DurableObjectExample extends DurableObject {
     // Test HTTP requests to container
     {
       let resp;
-      for (let i = 0; i < 6; i++) {
+      const maxRetries = 20;
```

Comment:
Seems high for a retry count. A comment about why `20` was selected would be helpful for context.